packages/venia-concept/src/components/Checkout/Receipt/asyncActions.js

```diff
@@ -0,0 +1,11 @@
+import qs from 'qs';
+import { getAccountInformation } from './selectors';
+import { resetCheckout } from 'src/actions/checkout/asyncActions';
+
+export const handleCreateAccount = history => async (dispatch, getState) => {
+    const accountInfo = getAccountInformation(getState());
+
+    await dispatch(resetCheckout());
+
+    history.push(`/create-account?${qs.stringify(accountInfo)}`);
```

Comment:
Instead of using the `qs` dependency to build a query string, please implement this with the new DOM standard [URLSearchParams](https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams). It is supported in all browsers we currently cover!

Reply:
Ok, I will do it, but as far as I can see IE does not support this interface https://caniuse.com/#search=URLSearchParams